src/main/java/org/folio/service/orders/flows/update/open/OpenCompositeOrderPieceService.java

```diff
@@ -168,10 +168,8 @@ public Future<Void> updatePieceRecord(Piece piece, RequestContext requestContext
   public Future<Void> openOrderUpdateInventory(CompositePoLine compPOL, Piece piece, boolean isInstanceMatchingDisabled, RequestContext requestContext) {
     if (Boolean.TRUE.equals(compPOL.getIsPackage())) {
       return titlesService.getTitleById(piece.getTitleId(), requestContext)
-        .compose(title -> inventoryInstanceManager.openOrderHandlePackageLineInstance(title, isInstanceMatchingDisabled, requestContext))
```

Comment:
If this method openOrderHandlePackageLineInstance() is not used in inventoryInstanceManager it can be deleted

Reply:
Removed it, forgot to remove after rebasing